src/components/layout/AppLogo.tsx

```diff
@@ -1,17 +1,16 @@
 import Image from 'next/image';
 
-interface AppLogoProps {}
+interface AppLogoProps {
+  theme: string | undefined;
+}
 
-const AppLogo: React.FC<AppLogoProps> = () => {
-  return (
-    <Image
-      priority
-      src="/portalo_dark.svg"
-      alt="Portalo"
-      width={120}
-      height={30}
-    />
-  );
+const AppLogo: React.FC<AppLogoProps> = ({ theme }) => {
+  const logoSrc =
+    theme === 'light'
+      ? '/assets/images/portalo_light2.svg'
```

Comment:
The logo crashed in light mode. It's a typo. 

```suggestion
      ? '/assets/images/portalo_light.svg'
```